src/app/components/pmapi/pmapi.service.js

```diff
@@ -228,10 +250,26 @@
 
             return deferred.promise;
         }
-
+      
+        function getMetricsMetadata(context) {
```

Comment:
This is not ideal, since we don't want to store all the information for `all` metrics, just the `unit`.